docs/going-into-production.rst

```diff
@@ -0,0 +1,175 @@
+.. _going-into-production:
+
+=====================
+Going into production
+=====================
+
+Running CrateDB in different environments requires different approaches. This
+document outlines the basics you need to consider when going into production.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _prod-auto-boot:
+
+Manual bootstrapping
+====================
+
+:ref:`Single host auto-bootstrapping <auto-bootstrapping>` is useful for
+development environments. However, auto-bootstrapping is typically too slow for
+production use. Additionally, for improved performance and resiliency,
+production CrateDB clusters should be run with one node per host machine. If
+you use multiple hosts, you must configure your cluster with :ref:`manual
+bootstrapping <manual-bootstrapping>`.
+
+
+.. _prod-config:
+
+Additional considerations
+=========================
+
+The following setting can be changed in your `configuration`_ file or set via
+command-line arguments. Additionally, you can change some settings at
+`runtime`_.
+
+
+.. _prod-config-path:
+
+Paths
+-----
+
+In a production environment, you should change the default locations of the
+data and log folders to avoid accidental deletion.
```

Comment:
Maybe add a note about disk sizing and disk IO?

Reply:
Are there any typical locations you would chose? Can we give a recommendation?